Reduce log verbosity in tests

martint opened the pull request:
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

findepi commented on core/trino-main/src/test/java/io/trino/execution/scheduler/faulttolerant/TestTaskDescriptorStorage.java:
```diff
@@ -53,6 +55,12 @@
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("org.hibernate.validator.internal.util.Version", Level.ERROR);
+        logging.setLevel("Bootstrap", Level.ERROR);
```
What is Bootstrap?

martint replied:
Dropped in b6cac03. The unnamed-`Bootstrap` configuration noise is now suppressed globally by `-Dairlift.quiet=true` (airlift/airlift#1667 + trinodb/trino#27884 wiring it into `air.test.jvm.additional-arguments.default`). The remaining static block keeps only the Hibernate Validator banner silencing, and `.quiet()` on `app.initialize()` is gone for the same reason.

findepi commented on core/trino-main/src/test/java/io/trino/server/protocol/spooling/TestPreferredQueryDataEncoderSelector.java:
```diff
@@ -24,6 +26,11 @@
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.server.protocol.spooling", Level.ERROR);
```
does this logger produce too much also in production?

cc @wendigo

martint replied:
Narrowed and lowered in b6cac03. The silencing is now scoped to `io.trino.server.protocol.spooling.QueryDataEncoders` only (not the whole package), at WARN. In this test path the noisy line is actually `QueryDataEncoders` emitting `"Spooling protocol is enabled with encodings: ..."` once per construction (5 tests × `new QueryDataEncoders(..., enabled=true)` = 5 INFO lines per run). At WARN, the `PreferredQueryDataEncoderSelector` WARN diagnostics the tests exercise (`"Requested one of the spooled encodings: ..., but these are only available: ..."`) still print, so we're not masking anything the tests depend on.

findepi commented:
Can this be simplified benefitting from
- https://github.com/airlift/airlift/pull/1667

?

martint committed:
Reduce log verbosity in tests

Test runs emit several startup banners and one-shot diagnostics that
are useful in production but pure noise in unit tests, where the same
constructors and modules are exercised hundreds of times per run.
Silence the noisiest sources for the affected tests and the testing
server. Prefer WARN over ERROR so genuine problems still surface.

martint commented on core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java:
```diff
@@ -158,8 +158,12 @@ public class TestingTrinoServer
         Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.connector.CatalogStoreManager", Level.ERROR);
+        logging.setLevel("io.trino.server.PluginManager", Level.ERROR);
+        logging.setLevel("io.trino.memory.RemoteNodeMemory", Level.ERROR);
```
Addressed in b6cac03 — switched the new silencings to INFO→WARN where INFO chatter is the issue (CatalogStoreManager, PluginManager, io.airlift.http.server). ERROR is kept only on RemoteNodeMemory, where the existing WARN messages ("Memory info update request to %s has not returned") are transient memory-info-fetch timing artifacts under test load, not actionable.

martint commented on plugin/trino-exchange-filesystem/src/test/java/io/trino/plugin/exchange/filesystem/s3/TestS3FileSystemExchangeManager.java:
```diff
@@ -26,6 +28,11 @@
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.bootstrap.exchange", Level.ERROR);
```
Dropped entirely in b6cac03 — all five `io.trino.bootstrap.exchange*` silencings are gone now that `-Dairlift.quiet=true` lands in `air.test.jvm.additional-arguments.default` (airlift/airlift#1667 merged 2025-12-04 and propagated via trinodb/trino#27884). The `TestAlluxioFileSystemExchangeManager` / `TestLocalFileSystemExchangeManager` / `TestS3FileSystemExchangeManager{,SseKms,SseS3}` files are no longer touched by the PR at all.

findepi commented on core/trino-main/src/test/java/io/trino/server/protocol/spooling/TestPreferredQueryDataEncoderSelector.java:
```diff
@@ -24,6 +26,11 @@
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.server.protocol.spooling.QueryDataEncoders", Level.WARN);
```
it's our own class. is it logging too verbosely?
